### PR TITLE
Adds extra paragraph and screenshot to further validate DCGM exporter

### PR DESCRIPTION
**Pull request by thomasvn**
## Related Issue

- Adds clarification which was required in [SUP-6416](https://kubecost.atlassian.net/browse/SUP-6416) and [SUP-6491](https://kubecost.atlassian.net/browse/SUP-6491)

## Proposed Changes

Specifically calls out the `DCGM_FI_PROF_GR_ENGINE_ACTIVE` metric.

[SUP-6416]: https://kubecost.atlassian.net/browse/SUP-6416?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SUP-6491]: https://kubecost.atlassian.net/browse/SUP-6491?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by mmurph3 — APPROVED** [review]
@thomasvn I think this is helpful and informative. Especially the prom query. This helped us with BAH.

File: install-and-configure/advanced-configuration/gpu.md [hunk 1]
```diff
@@ -349,6 +349,10 @@ Open the Prometheus web interface in your browser by navigating to `http://local
 
 ![Prometheus query showing DCGM Exporter metric](/images/gpu-prometheus-query.png)
 
+Additionally, check the `DCGM_FI_PROF_GR_ENGINE_ACTIVE` metric. This is the metric Kubecost uses to determine GPU utilization. GPU efficiency features in the UI are only enabled when there are nonzero values for this metric.
```
Comment (chipzoller) [inline install-and-configure/advanced-configuration/gpu.md:352]:
```suggestion
Additionally, check the `DCGM_FI_PROF_GR_ENGINE_ACTIVE` metric. This is the metric Kubecost currently uses to determine GPU utilization. GPU efficiency features in the UI are only enabled when there are non-zero values for this metric.
```

**Review by chipzoller — COMMENTED** [review]
LGTM. Minor suggestion.

**Comment by thomasvn** [comment]
Link checker seems to be flakey for https://cloud.google.com links right now. I will try to re-run it tomorrow.

```
 ## Summary

| Status        | Count |
|---------------|-------|
| 🔍 Total      | 1350  |
| ✅ Successful | 1297  |
| ⏳ Timeouts   | 0     |
| 🔀 Redirected | 0     |
| 👻 Excluded   | 47    |
| ❓ Unknown    | 0     |
| 🚫 Errors     | 3     |

## Errors per input

### Errors in install-and-configure/install/cloud-integration/gcp-out-of-cluster/README.md

* [500] [https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables#detailed-usage-cost-data-schema](https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables#detailed-usage-cost-data-schema) | Failed: Network error: Internal Server Error

### Errors in install-and-configure/advanced-configuration/gpu.md

* [500] [https://cloud.google.com/kubernetes-engine/docs/how-to/dcgm-metrics](https://cloud.google.com/kubernetes-engine/docs/how-to/dcgm-metrics) | Failed: Network error: Internal Server Error

### Errors in kubecost-cloud/kubecost-cloud-cloud-billing-integrations/kubecost-cloud-gcp-integration.md

* [500] [https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables#detailed-usage-cost-data-schema](https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables#detailed-usage-cost-data-schema) | Cached: Error (cached)
```

**Comment by thomasvn** [comment]
Link checker passed this time. Re-requesting reviews.

**Pull request merged**